docs/admin/addons.md

```diff
@@ -0,0 +1,24 @@
+---
+---
+
+## Overview
+
+Add-ons extend the functionality of Kubernetes in a pluggable way.
+
+This page lists some of the available add-ons and links to their respective installation instructions.
+
+## Networking and Network Policy
```

Comment:
Honestly, reading this now all three of these still just feel like marketing speak, and I really don't want this to be a marketing page - especially as more solutions get added to the list.  I think this has a lot of potential to become a "my technology is better than your technology" fest.  Still feel pretty strongly that the one sentence descriptions _do not belong here_.

On a secondary note, I had thought lists like this should be in alphabetical order. I understand that puts Calico at the top, but the intention is just so that some level of order is maintained rather than everyone scrambling for the top.  An alternative that puts Weave at the top would be to use reverse alphabetical order :)